<a href="https://colab.research.google.com/github/sabinamunoz/lab-t-tests-p-values/blob/master/lab_t_tests_p_values.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#**1.**

In [1]:
import statistics
import math
import numpy as np
from scipy.stats import ttest_ind, norm

In [2]:
new_machine_tests = [42.1, 41, 41.3, 41.8, 42.4, 42.8, 43.2, 42.3, 41.8, 42.7]
old_machine_tests = [42.7, 43.6, 43.8, 43.3, 42.5, 43.5, 43.1, 41.7, 44, 44.1]

new_machine_mean = statistics.mean(new_machine_tests)
old_machine_mean = statistics.mean(old_machine_tests)
print(new_machine_mean)
print(old_machine_mean)

42.14
43.230000000000004


- H0: u = 43.23


- Ha: u < 43.23

In [3]:
pop_mean = 43.23

n = 10
sample_mean = 42.14
sample_std = np.std(new_machine_tests)

statistic = (sample_mean - pop_mean)/(sample_std/math.sqrt(n))
statistic

-5.316124512154909

**T-table: one-tailed alpha 0.05 9DF = 1.833**


- abs(-5.316) > 1.66 - null hypothesis rejected with 95% confidence


**Results show a statistical significance in the average packing time of the two machines:**

- the new machine is faster